runtime/router/router.go

```diff
@@ -64,6 +64,10 @@ type Router struct {
 	// unrecovered panics.
 	PanicHandler func(http.ResponseWriter, *http.Request, interface{})
 
+	// Used for special behavior using which different handlers can configured
+	// for paths such as /a and /:b in router.
+	WhitelistedPaths []string
```

Comment:
update the comment at the top of router comment.

Reply:
Sure